certbot/cli.py

```diff
@@ -570,6 +570,8 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
               "additional command line flags; the client will try to explain "
               "which ones are required if it finds one missing")
     helpful.add(
+        None, "--gui", dest="gui", action="store_true", help="Run using GUI")
```

Comment:
This should probably be `--dialog` not `--gui`. `dialog` is questionably a GUI and this allows us to add another UI later if we want to.